node-src/lib/checkPackageJson.ts

```diff
@@ -22,7 +22,7 @@ export default async function checkPackageJson({
   if (!options.interactive) return;
 
   try {
-    const json = { ...packageJson };
+    const { readme, _id, ...json } = packageJson;
```

Comment:
These fields (`readme` & `_id`) are added to package.json if we don't remove them here.